src/UBAFeeCalculator/UBAFeeCalculator.ts

```diff
@@ -20,7 +20,7 @@ export default class UBAFeeCalculator {
     private readonly config: UBAConfig,
     private readonly logger: Logger,
     protected readonly originSpoke: UBASpokeBalanceType,
-    protected readonly destinationSpoke: UBASpokeBalanceType
+    protected readonly refundSpoke: UBASpokeBalanceType
```

Comment:
I thought we didn't want to store the refund spoke in the calculator?

I was thinking instead each calculator had a single chain ID for which it provides functions to compute deposit and refund fees for?

Reply:
@nicholaspai Yes, this is just the refactor - the next PR will be for moving this outside of the class